src/directives/btngroup.js

```diff
@@ -28,6 +28,7 @@ goog.require('ngeo');
  *
  *     <div ngeo-btn-group ngeo-btn-group-active="ctrl.drawToolActive">
  *
+ * @htmlAttribute {???} ngeo-btn-group-active ???
```

Comment:
@htmlAttribute {*} ngeo-btn-group-active Any property of the scope. Tells whether at least one button of the group is active.